build.rs

```diff
@@ -0,0 +1,11 @@
+fn main() {
+    // Setup cfg aliases
+    cfg_aliases::cfg_aliases! {
+        // Vendors/systems
+        wasm: { target_arch = "wasm32" },
```

Comment:
Because we only use these in a few places, maybe we could just write them inline instead of having build.rs with aliases

Reply:
I thought `#[cfg(all(target_arch = "wasm32", not(feature = "webgl")))]` was not very explicit for WebGPU cases.
Let me know if you still want me to remove this.

Reply:
Maybe we could avoid explicit checks for WebGPU because it can always be included (i.e. try to use WebGPU first, then WebGL if the feature is enabled)

Reply:
Outch, it seems that Wgpu-rs has not been designed to permit this …
backend/mod.rs is clearly a toggle between WebGPU backend and others, so making a build enabling WebGPU AND another backend is kind of a big rework …